keps/sig-cli/3104-introduce-kuberc/kep.yaml

```diff
@@ -23,17 +25,20 @@ stage: alpha
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.25"
+latest-milestone: "v1.31"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
-  alpha: "v1.25"
-  beta: "v1.27"
-  stable: "v1.28"
+  alpha: "v1.31"
+  beta: ""
+  stable: ""
 
 # The following PRR answers are required at alpha release
 # List the feature gate name and the components for which it must be enabled
-feature-gates: []
+feature-gates:
+  - name: KUBECTL_KUBERC
+    components:
+      - kubectl
```

Comment:
I think this is an environment variable, not a feature gate.

I'd like feature gates to only mean https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/; if we want kubectl feature flags listed there, let's first change that page to list them, or put up a separate page (the two pages could link to each other).

Reply:
The [kep.yaml template](https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/kep.yaml) doesn't have an option for other, like we have in [main document](https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md#feature-enablement-and-rollback). I don't want to break automation build around that bit :sweat_smile: